pages/common/doxygen.md

```diff
@@ -0,0 +1,11 @@
+# doxygen
+
+> A documentation system for various programming languages.
+
+- Generate a template configuration file named configName:
+
+`doxygen -g {{configName}}`
```

Comment:
Change configName => path/to/config_file

Reply:
Yeah, You're right. `Cut off unnecessary information.`